client/lib/plans/constants.js

```diff
@@ -262,9 +262,9 @@ export const PLANS_LIST = {
 
 	[ PLAN_BUSINESS ]: {
 		getTitle: () => i18n.translate( 'Business' ),
-		getAudience: () => 'Best for small businesses', //PLANS A/B TEST: Translate if test passes
-		getBlogAudience: () => 'Best for brands', //PLANS A/B TEST: Translate if test passes
-		getPortfolioAudience: () => 'Best for small businesses', //PLANS A/B TEST: Translate if test passes
+		getAudience: () => i18n.translate( 'Best for small businesses' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Best for Small Business:' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).